collect_app/src/main/java/org/odk/collect/android/utilities/FileUtils.java

```diff
@@ -504,4 +519,26 @@ public static Bitmap getBitmap(String path, BitmapFactory.Options originalOption
 
         return bitmap;
     }
+
+    public static String readFile(File file) {
```

Comment:
Did you consider not hex encoding the digests, but simply writing the byte arrays as binary files?

Reply:
No I didn't but seems like a great idea! I will definitely try it out